test/unittests/execute_floating_point_test.cpp

```diff
@@ -581,6 +583,54 @@ TYPED_TEST(execute_floating_point_types, div)
     EXPECT_THAT(exec(TypeParam{0xABCD.01p7}, TypeParam{4}), Result(TypeParam{0x1.579A02p20}));
 }
 
+TYPED_TEST(execute_floating_point_types, min)
+{
+    auto instance = instantiate(parse(this->get_binop_code(Instr::f32_min)));
+
+    // Check every pair from cartesian product of the list of values.
+    for (size_t i = 0; i < std::size(this->ordered_special_values); ++i)
```

Comment:
While using this list is nice, I think having explicit test cases based on the spec (in the opening commit) would be useful.

Reply:
Duplicating the same checks is less confusing?

Reply:
I'm not sold on this list, it is a pain looking it up every single time.

Reply:
Done.